OmniBLE/Bluetooth/Packet/BLEPacket.swift

```diff
@@ -154,7 +154,7 @@ struct LastOptionalPlusOneBlePacket: BlePacket {
     let size: UInt8
 
     func toData() -> Data {
```

Comment:
I changed this code to match up with how the Kotlin code adds the padding and data.

```
    override fun toByteArray(): ByteArray {
        return byteArrayOf(index, size) + payload + ByteArray(MAX_SIZE - payload.size - 2)
    }
```